workos/organizations.py

```diff
@@ -88,14 +88,23 @@ def create_organization(self, organization):
                     within the Organization allow profiles that are outside of the Organization's
                     configured User Email Domains. (Optional)
                 organization[domains] (list) - List of domains that belong to the organization
+            create_organization_options (dict) - Options configuration for creating an organization. (Optional)
```

Comment:
Can we model this more after the Audit Logs implementation? https://github.com/workos/workos-python/blob/main/workos/audit_logs.py#L23

Key differences:
- We can just pass an `idempotency_key` parameter with a default value of `None`
- Construct the header like:
```
 headers = {}
        if idempotency_key:
            headers["idempotency-key"] = idempotency_key
```
- pass those `headers` into the request method

Reply:
Updated ✅